service-bus/bus/Cargo.toml

```
@@ -11,8 +15,8 @@ json=["serde_json"]
 msgpack=["rmp-serde"]
 
 [dependencies]
-ya-sb-proto = "0.1"
-ya-sb-util = "0.1"
+ya-sb-proto = { version = "0.1", path="../proto" }
+ya-sb-util = {version = "0.1", path="../util", features=["with-futures"] }
```

Is it needed while we have crates.io patched in top-level `Cargo.toml`?

If yes, pls add space
```suggestion
ya-sb-util = { version = "0.1", path="../util", features=["with-futures"] }
```